app/javascript/dashboard/routes/dashboard/settings/profile/MessageSignature.vue

```diff
@@ -1,40 +1,40 @@
 <template>
-  <form @submit.prevent="updateSignature()">
-    <div class="profile--settings--row text-black-900 dark:text-slate-300 row">
-      <div class="w-[25%] py-4 pr-6 ml-0">
-        <h4 class="block-title text-black-900 dark:text-slate-200">
-          {{ $t('PROFILE_SETTINGS.FORM.MESSAGE_SIGNATURE_SECTION.TITLE') }}
-        </h4>
-        <p>{{ $t('PROFILE_SETTINGS.FORM.MESSAGE_SIGNATURE_SECTION.NOTE') }}</p>
-      </div>
-      <div class="p-4 w-[45%]">
-        <div>
-          <label for="message-signature-input">{{
-            $t('PROFILE_SETTINGS.FORM.MESSAGE_SIGNATURE.LABEL')
-          }}</label>
-          <woot-message-editor
-            id="message-signature-input"
-            v-model="messageSignature"
-            class="message-editor h-[10rem]"
-            :is-format-mode="true"
-            :placeholder="
-              $t('PROFILE_SETTINGS.FORM.MESSAGE_SIGNATURE.PLACEHOLDER')
-            "
-            :enabled-menu-options="customEditorMenuList"
-            :enable-suggestions="false"
-            @blur="$v.messageSignature.$touch"
-          />
-        </div>
-        <woot-button
-          :is-loading="isUpdating"
-          type="submit"
-          :is-disabled="$v.messageSignature.$invalid"
-        >
-          {{ $t('PROFILE_SETTINGS.FORM.MESSAGE_SIGNATURE_SECTION.BTN_TEXT') }}
-        </woot-button>
+  <div class="profile--settings--row text-black-900 dark:text-slate-300 row">
```

Comment:
Any particular reason to remove the `form`?

Reply:
@scmmishra Yes, I removed it because I face some weird issues, like saving the signature when we click inside the editor.